src/Tools/Extensions.ApiDescription.Client/src/build/Microsoft.Extensions.ApiDescription.Client.targets

```diff
@@ -44,7 +43,7 @@
         BuildInParallel="$(BuildInParallel)"
         Projects="@(OpenApiProjectReference)"
         RebaseOutputs="true"
-        RemoveProperties="Configuration;Platform;RuntimeIdentifier;TargetFramework;TargetFrameworks">
+        RemoveProperties="Configuration;Platform;RuntimeIdentifier;TargetFrameworks;%(OpenApiProjectReference.GlobalPropertiesToRemove)">
```

Comment:
Do you recall why `TargetFrameworks` is in this list? It should be harmless, but I don't know of a case where it's passed as a global property. Do I need to update my mental map on that?

Reply:
Thought I noticed `TargetFrameworks` was defined in inner builds. But, if that's just coming from the setting in the project file and won't be passed along to the referenced projects, I can remove it.

Given your comments above about `Configuration` and `Platform` (well, `%(ShouldUnsetParentConfigurationAndPlatform)`), should I _remove_ `Configuration;Platform;` here?

Reply:
Yes, you should, great catch!

Reply:
Also, no need to bother removing TargetFrameworks. It should be harmless, so don't change it just before release. But my understanding is that while it will be set during both inner and outer builds for a multitargeted project, it should only be as a normal property, not global.

Reply:
OK, will make the first change